### ISSUE TITLE
New proposal Website

### ISSUE THREAD

**Issue by tkm999** [turtlecoin/meta#117]
I have designed a draft prototype for the website and it is not final. I collected contents from the current website turtlecoin.lol I will be adding SPECs section later after collecting feedback. 

I need your feedback on words, design, content or anything. Your feedback is greatly appreciated.

To be a great, attractive and useful website, it does need improvements on the contents and re-word them from the current website. Some of the contents are good while others are plain and nothing unique. In general, I think descriptions of technical use is the most important part for developers and crypto fans to get to know about Turtlecoin

**For example:** 

1) Reword the explanation of **Privacy, community driven, Easy to use, Easy to mine, ..etc** because too many cryptos are doing same thing. 
             For **"Privacy"**, what technology does Turtle use? How this is different from 
             others? 
        **"Easy to use"**, What does it mean? Do you mean the wallets are accessible for everyone? If  
            elaborate a bit more.
          I believe Turtle somewhat has an unique but I would like the community to express.

2) In general, how Turtlecoin is better than others? what does it make different? 
      (This is an important part because I will emphasize this in a headline where people are looking 
       for something different) 

Any feedback, suggestion or ideas? Please feel free to comment. I would be gladly to help to build the website. When it is done, I will push the project to github so people can work on it if needed.

![1](https://user-images.githubusercontent.com/40526750/41940713-b7a2c7d6-7967-11e8-8c30-722b9c3965d1.png)
![2](https://user-images.githubusercontent.com/40526750/41940754-d2bf51c4-7967-11e8-8455-e200641fffff.png)
![3](https://user-images.githubusercontent.com/40526750/41940759-d7953ff6-7967-11e8-964b-55a64e57ffe9.png)
![4](https://user-images.githubusercontent.com/40526750/41940760-d7a58532-7967-11e8-9005-98d403297b44.png)
![5](https://user-images.githubusercontent.com/40526750/41940761-d7b1641a-7967-11e8-9e57-83c87f27f1c8.png)
![6](https://user-images.githubusercontent.com/40526750/41940763-d7bc7828-7967-11e8-8ba7-73864ae7caf6.png)

**Comment by ar-x** [comment]
Unrelated - https://github.com/turtlecoin/meta/issues/98 could use a designer to mockup the docs frontend :) cc @SoreGums

**Comment by njr1081** [comment]
I like the changes to the wallet, info and contact sections. I think the design could benefit from a different picture on the initial page as well as sticking to a singular theme (it switches from light to dark for example).

**Comment by SoreGums** [comment]
http://turtlewiki.netlify.com > is sajo-docs

It is a manual thing the builds by copy / pasting info from the docs repo [turtlecoin/turtlecoin-docs](https://github.com/turtlecoin/turtlecoin-docs).

It is not the official frontend for the docs, it is a placeholder and a project by a community member.
docs is a lot of things and that site only covers the user guides part.

--

_since this is design, I am blunt and direct, design is subjective and personal, these are my thoughts_

Regarding your design.
It looks nice - however content wise there are issues.

### 1) GeoIP
The GeoIP background is pretty cool though on the current site.
Considering the central theme of the project, `decentralised trust network`, the whole _here are all the nodes that make up that network_ is a thing we'd like to make obvious.

### 2) The story
This is also a central theme to the project and important. This is why the project exists and is the basis from which the community operates the way it does.

### 3) Features
The icons are needed, provides contrast and something about the brain and reading and pschoogy, NLP stuff. I can't explain it, just needs to be there.

### 4) The fine print
This is needed, sure it is fluff, however its a strong message about the project accross realy fast and simply. Important for new people.

OK that's enough.

In summary, my experience of your idea is _Simple & Generic_
This is not the best impression.

Maybe you could assist us with the Docs frontend #98 ?
Thanks for giving this a go, I know my comments are harsh, I had a go at softening them, however I failed at that as well :/

**Comment by kupustaa** [comment]
Hey Khem Boi here. To me, the design is nice, but this has lost some really inviting flare the original site has.

Well for design, it would be cool to have it raining turtles because we always make it rain, but what got to me was the road map, the community driven section, and join section.

First off, where's the story? That was the best part of the website. Sure it was kinda memey and comical but that's what got me and probably a lot other users involved, it added humility to the project. It made it stand out in the sense these aren't just nerds programming, they're real life people trying to do something different. It was very memorable as well, I've been on the internet all my life and I can tell you I will remember that introduction, but I can't even remember the last bootstrap website I was just on.

Second, the road map. Before it was simple and gave a sense of mystery, not too much given out but enough to make the user go hmm I want to read their code. If you're going to write about smart contract's, it should be written in a way that shows what the community is doing different, not that we are keeping up with industry standard. That's how I take all the road map but mainly the smart contract area grinded my gears for some reason because it doesn't make the project stand out, it gives the user not enough and too much info compared to before.

Third, the community driven section. It feels lame, they don't yell at users? That's to be expected in general, even though that's sadly not true with a lot of projects in the industry, how that lil box is worded it makes me feel like I will get yelled at. That's just my prespective as a customer/passerby. There shouldn't be a please join at the end of that. People will join if they want to. For the other boxes, mining should include the algorithm, and the easy get started shouldn't say how easy it is, it should make it easy to get started!

My last and final impression, this was the selling point for me, is your closing page. It's a bit too generic, it's not welcoming all users like the original one. It gives the impression "I gotta know block chain/Bitcoin to be of value to this community" not "I don't care who you are get in here and make yourself at home". It closes doors. The closing end of the original made me think wow, even though I haven't programmed in years, I can somehow use my chemistry brain to help this industry. It invited everyone from any background.

I think what was magical about the first site is that it was simple, unique, and didn't have to say anyone can do this, it just showed we want anyone to be able to do this. I hope all this helps you with design, sorry if any of this was harsh at all. I really like your design skills. It's just missing that magic touch friend, good luck.

**Comment by SoreGums** [comment]
nothing for now - can re-open if needed
